multibase-conv/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	multibase "github.com/multiformats/go-multibase"
+)
+
+func main() {
+	if len(os.Args) < 3 {
+		fmt.Printf("usage: %s NEW-BASE CID...\n", os.Args[0])
```

Comment:
> definitely not usual to use uppercase.

It is an older convention used by many unix utilities.  I agree the `<>` is better and we should go with that since that is what IPFS uses.